### DISCUSSION TITLE
Any way to download 64-bit tools like cmake

### DISCUSSION THREAD

**Discussion by playgithub** [microsoft/vcpkg#17865] [Ideas]
For example,
by default cmake-3.18.4-win32-x86.zip will be downloaded,
any way to download cmake-3.18.4-win64-x64.zip instead?

**Comment by JackBoosY** [comment]
Good idea.

**Comment by playgithub** [comment]
@NancyLi1013 
Is it solved?

**Reply by NancyLi1013** [reply]
No, it hasn't been added now.
